com.dreamteam.database/src/main/resources/application.properties

```diff
@@ -0,0 +1 @@
+
```

Comment:
What is this file?

Reply:
its part of the webservices class for when we add them later on.